### DISCUSSION TITLE
LauchBase da rockseat é bom? vale a pena?

### DISCUSSION THREAD

**Discussion by rafassilveira** [frontendbr/forum#1659] [Discussão]
Olá galera,
Minha esposa está começando programar, e ela tá procurando um curso legal online p, eu vi que o gostask da rocketseat seria muito avançado pra ela, porém vi esse LauchBase que lançaram ano passado, algu´me já fez esse curso? poderia dizer como foi a experiência?
vlwww :)

**Comment by gmtborges** [comment]
Tenho o bootcamp e é muito bom! Esse não conheço, mas posso indicar o Guia da freeCodeCamp

[https://www.freecodecamp.org/learn](https://www.freecodecamp.org/learn)

**Comment by rafassilveira** [comment]
> Tenho o bootcamp e é muito bom! Esse não conheço, mas posso indicar o Guia da freeCodeCamp
> 
> https://www.freecodecamp.org/learn

Sou muito fan do FCC, porém ela nao entende muito inglês ainda. :)

**Comment by wilsonneto-swe** [comment]
tenho e fiz o bootcamp da rocketseat, os starters deles e o curso do dev pleno.

Para iniciantes, eu senti a pegado do curso do Dev Pleno bem mais fácil de assimilar, mas recomendo começar por ele e fazer o bootcamp também... Claro que vendo a documentação efazendo alguns projetos/testes próprios tbm.

O curso do dev pleno é: http://bit.ly/reactjs-dev-pleno

O custo dele é bem mais tranquilo que o bootcamp, mas ambos são muito bons.

Abraços!
